guides/v2.3/graphql/mutations/add-virtual-products.md

```diff
@@ -127,3 +127,14 @@ Attribute |  Data Type | Description
 {% include graphql/cart-object.md %}
 
 [Cart query output]({{page.baseurl}}/graphql/queries/cart.html#cart-output) provides more information about the `Cart` object.
+
+## Errors
+
+Error | Description
+--- | ---
+`Could not find a cart with ID "XXX"` | The specified `cart_id` value does not exist in the `quote_id_mask` table.
+`Could not find a product with SKU "YYY"` | A simple product with the SKU specified in the `data`.`sku` attribute does not exist.
```

Comment:
@atwixfirster Should this say "A virtual product"?

Reply:
should....

**fixed**

Thank you, @keharper